### PR TITLE
add Info in docs

### PR DESCRIPTION
**Pull request by williamrozin**

### CODE REVIEW

**Review by cuchi — APPROVED** [review]
🔝

**Comment by haskellcamargo** [comment]
The documentation has been migrated to the branch [`mkdocs`](https://github.com/rung-tools/rung-cli-docs/tree/mkdocs) and is available [now in this site](https://rung-tools.github.io/rung-cli-docs).

File: features/info.rst [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.. _resources:
+
+=========
+Informações adicionais
+=========
```
Comment (haskellcamargo) [inline features/info.rst]: `=========` should have length of the text

File: features/info.rst [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+Atualmente o Rung possibilita que o desenvolvedor defina, além da descrição, um conteúdo adicional (no formato markdown) a ser apresentado na página da sua extensão na loja. Este conteúdo adicional é opcional, mas é recomendado o seu uso de tal forma que o seu conteúdo instigue o usuário a instalar e utilizar a extensão.
```
Comment (haskellcamargo) [inline features/info.rst]: > *Atualmente, o Rung*

**Comment by williamrozin** [comment]
fixed

**Review by larissathasdefar — APPROVED** [review]

**Review by haskellcamargo — APPROVED** [review]

**Pull request merged**